nepi/main/tests/test_templatetags.py

```diff
@@ -170,10 +181,10 @@ def test_average_session_scores(self):
                  self.userBadScore, self.userGoodScore]
 
         ctx = average_session_score(users, self.hierarchy)
-        self.assertEquals(ctx['sessions'][0], 67)
+        self.assertEquals(ctx['sessions'][0], 75)
         self.assertEquals(ctx['sessions'][1], None)
         self.assertEquals(ctx['sessions'][2], None)
-        self.assertEquals(ctx['average_score'], 67.0)
+        self.assertEquals(ctx['average_score'], 75.0)
```

Comment:
Comparing floats with "equals" logic is dangerous. This particular case might work, but you could experience failures on different platforms/python versions, etc. You want to use `assertAlmostEqual` instead: https://docs.python.org/2/library/unittest.html#unittest.TestCase.assertAlmostEqual